src/cloud-api-adaptor/podvm/Dockerfile.podvm_binaries.fedora

```diff
@@ -25,7 +25,7 @@ ARG PAUSE_REPO
 ARG PAUSE_VERSION
 ARG PAUSE_BIN
 ARG IMAGE_NAME
-ARG GH_TOKEN
```

Comment:
nit: I guess we could remove the `GH_TOKEN` refs from commit 1 as we just remove them here?